docs/reference/upgrade.md

```diff
@@ -22,12 +22,14 @@ Starting with version <TBD.X.X>, lakeFS has transitioned from using a PostgreSQL
 multiple database implementations. More information can be found [here](link-to-lakefs-on-kv-documentation).  
 Users upgrading from a previous version of lakeFS must pass through the KV migration version (<TBD.X.X>) before upgrading to newer versions of lakeFS.
 
-> **Note:**  
-> **Users using OS environment variables for database configuration must define the `connection_string` explicitly or as environment variable before proceeding with the migration.**  
-> 
-> Prior to the migration procedure it is strongly recommended to perform the following steps:
-> * Commit all uncommitted data on branches
-> * Create a snapshot of your database
+> **IMPORTANT: Pre Migrate Requirements**  
+> * **Users using OS environment variables for database configuration must define the `connection_string` explicitly or as environment variable before proceeding with the migration.**  
+> * **Database storage free capacity of at least twice the amount of the currently used capacity**
```

Comment:
This sentence is a bit vague:
Which DB needs the free space? Is it the original DB that needs the extra space?

Reply:
In addition, why is it `twice`?

Reply:
Removed redundant.
We don't define the user which DB he should use. The user can either use the current one or create a new one.

Reply:
I don't have any better value. I was just wondering if the 2x is really require or it is an overkill. If a user has huge system, with an enormous DB - is requiring an additional 2x (Did I understand it correctly? It is **additional** twice the capacity?) a realistic demand?

Reply:
It's twice the currently used capacity, not the allocated capacity. For example, if the user is currently uses 2GB out of 10GB, he does not need to allocate more storage.
On the other hand if he uses 9GB out of 10, he will need to allocate 9GB more (assuming that he's going to keep using the server and his data will grow - it makes sense to me no?)

Reply:
So, in that case I would remove the "twice", as it says (at least as I understood it) that the free capacity should be at least twice than the used one. In that case it would imply an additional 17GB to have 18GB free

Reply:
Yes - that's what I meant (miscalculation), additional 18GB. If the user has already 9 GB of data it means that in the end of the migration he will have at least 18 GB of data therefore increasing capacity to 19GB makes no sense